.travis.yml

```diff
@@ -52,13 +52,13 @@ jobs:
   - env: TEST_TYPE=upgrade CASSANDRA_VERSION=git:trunk CO="-t ~@all_nodes_reachable"
   include:
     - stage: Integration Tests
-      name: Memory, H2 and Postgresql
+      name: Cassandra 1.2.19: memory, H2 and Postgresql
```

Comment:
to nitpick: would this read better in the format: `Memory, H2, and Postgresql (repairing Cassandra 1.2.19)` ?
it's awkward… i'm not sure how to best distinguish between what storage versus what c* version we're testing against…

Reply:
Or `Memory, H2 and Postgresql on Cassandra 1.2.19` to better match the other tests like `Two Reapers on Cassandra 2.1.20` and `Four Flapping Reaper on Cassandra 2.2.13`?